changelog.mdx

```diff
@@ -4,6 +4,58 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
 ---
 
+## 2024-06-20
+
+- We merged and simplified test case view. Bye bye, way too many tabs.
+  <Frame caption="merged test case view, screenshot 06/2024">
+    <img
+      src="/images/changelog/test-case-view-new.png"
+      alt="merged test case view"
+    />
+  </Frame>
+
+## 2024-06-17
+
+- `shadow DOM and iFrames` - the AI agent can hangle most shadow DOMs and iFrames now
+
+## 2024-06-13
+
+- We mark broken locators now. A yellow highlight and an indicator will point to the locator that doesn’t do the thing it’s supposed to.
+  <Frame caption="screen recording of a broken locator being pinpointed, 06/2024">
+    <img
+      src="/images/changelog/broken-locator.gif"
+      alt="pinpointing broken locators"
+    />
+  </Frame>
+
+## 2024-06-10
+
+- We removed automatically generated `link tests`. We generated too many link tests on dynamic elements that broke constantly. We will come back with a better solution.
+
+## 2024-06-05
+
+- We add `dependencies` automatically now. We now add a cookie banner test as dependency automatically. We will be adding more common dependencies in the future.
+- less obstructive `test editing` - interaction and assertion types can be switched now
+
+## 2024-06-04
+
+- `AI discovery`: we automatically run, generate steps and code, and validate all successfully discovered test cases and publish them to your test suite.
```

Comment:
think it'd be nice to contrast this, rough suggestion: instead of just giving you proposals for test cases, we now generate steps and validate the proposed test cases automatically. Tests we feel confident about are published to your test suite automatically.

Reply:
```suggestion
- `AI discovery`: Instead of just giving you proposals for test cases, we now generate steps and validate the proposed test cases automatically. Tests we feel confident about are published to your test suite automatically.
```